### ISSUE TITLE
Book download incomplete?

### ISSUE THREAD

**Issue by polanar** [openaudible/openaudible#390]
Downloading a single book from audible US, using a macbook. The book appears to download and shows as complete in openaudible, but the file remains as .aax.part and the "convert" option is greyed-out.

**Comment by openaudible** [comment]
Open the Console (CMD-E) and try again. Watch the log for any errors. Feel free to report back. (Please remove the download link URL if you paste the log). Make sure the location it is saving to has enough disk space.

**Comment by polanar** [comment]
And this time it worked. No error message, book downloaded and then offered to convert. I'd tried to download three times earlier today, plus at least ten times about three weeks ago, with the same failure every time. The only difference between this file and all my other audible books (all of which downloaded without any difficulty) is that this one is from .com rather than .co.uk or .ca. Save location has >160gb free and has had at least that space all along. I've no idea why it worked this time, unless the software has a built-in "dammit, they've got the console open, they must mean business" feature...

One other interesting thing: when I tried to submit a bug report three weeks ago, and again this morning with the most up-to-date version, I got an error message. Just tried it again, and it worked too. Can't explain that one either! 

Thanks for the assistance anyway though!
